instant.ts

```diff
@@ -26,11 +26,17 @@ interface PackagesMap {
   [packageID: string]: PackageInfo
 }
 
+type EnvironmentVars = Array<{
```

Comment:
It seems a bit weird that we're including the Array here as part of the type. I would think the type should be singular and then when you declare it on line 142, you should have it as `const envVars = [] as Array<EnvironmentVar>`.

Reply:
nice, done:)